signature/spec/README.md

```diff
@@ -0,0 +1,118 @@
+# OCI Image Signature Specification
+
+**Version 0.1**
+
+## Introduction
+
+This document specifies an out-of-band cryptographic verification model for
+container images. Detached signatures are associated with images by manifest digest hash value.
+
+## Signature Format
+
+### Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="critical"></a>critical | [ [CriticalObject](#criticalObject) ] | **Required.**
+<a name="optional"></a>optional | [ [OptionalObject](#optionalObject) ] | **Optional.**
+
+#### <a name="criticalObject"></a>Critical Object
+
+Field Name | Type | Description
+---|:---:|---
+<a name="identity"></a>identity | [ [IdentityObject](#identityObject) ] | **Required.**
+<a name="image"></a>image | [ [ImageObject](#imageObject) ] | **Required.**
+<a name="type"></a>type | `string` | **Required.** Valid values: "atomic container signature"
```

Comment:
What other sorts of things do you imagine putting in the `type` field?  In my signed name-assertion proposal (opencontainers/image-spec#176), I suggest just `name` and `blob` fields (close to your `identity` and `image` fields).  Then you can use the object for naming any blob (not necessarily a manifest(-list)).  Folks consuming the name-assertions would know them as such because the descriptor referencing them has `mediaType` set to `application/vnd.oci.image.named.blob.v1+json`.  Embedding type information in the blob itself (like your `type`?) gives a chicken/egg problem of “which parser to I use on this blob if I have to parse it to find the type and I need the type to pick a parser”.